src/Api/PubnubApi/EndPoint/Access/GrantOperation.cs

```diff
@@ -120,6 +120,19 @@ public void Execute(PNCallback<PNAccessManagerGrantResult> callback)
 #endif
         }
 
+        public async Task<PNResult<PNAccessManagerGrantResult>> ExecuteAsync()
+        {
+            if (config != null && pubnubLog != null)
+            {
+                LoggingMethod.WriteToLog(pubnubLog, string.Format("DateTime: {0}, WARNING: Grant() signature has changed! This specific call will be making a request to PAMv2. Please update your code if this is not the intended action.", DateTime.Now.ToString(CultureInfo.InvariantCulture)), config.LogVerbosity);
+            }
+#if NETFX_CORE || WINDOWS_UWP || UAP || NETSTANDARD10 || NETSTANDARD11 || NETSTANDARD12
+            return await GrantAccess(this.pubnubChannelNames, this.pubnubChannelGroupNames, this.pamAuthenticationKeys, this.grantRead, this.grantWrite, this.grantDelete, this.grantManage, this.grantTTL, this.queryParam).ConfigureAwait(false);
+#else
+            return await GrantAccess(this.pubnubChannelNames, this.pubnubChannelGroupNames, this.pamAuthenticationKeys, this.grantRead, this.grantWrite, this.grantDelete, this.grantManage, this.grantTTL, this.queryParam).ConfigureAwait(false);
```

Comment:
This `else` also looks the same as the `if` maybe something in C# that i'm not understanding?

Reply:
will remove if else